src/main/java/nextstep/ladder/domain/position/Position.java

```diff
@@ -0,0 +1,71 @@
+package nextstep.ladder.domain.position;
+
+import java.util.Objects;
+
+import static nextstep.ladder.domain.position.Direction.LEFT;
+import static nextstep.ladder.utils.Validator.checkNotNull;
+
+public class Position {
+
+    public static final String NOT_LESS_THAN_ERROR_MESSAGE = " 보다 작지 않습니다.";
+    public static final String NEGATIVE_ERROR_MESSAGE = "음수는 불가능합니다";
+    private static final int MIN_VALUE = 0;
+    private static final int VALUE_UNIT = 1;
+
+    private final int value;
+
+    public Position(int value) {
+        checkNotNegative(value);
+        this.value = value;
+    }
+
+    private void checkNotNegative(int position) {
+        if (position < MIN_VALUE) {
+            throw new IllegalArgumentException(NEGATIVE_ERROR_MESSAGE);
+        }
+    }
+
+    public void checkLessThan(int other) {
+        if (value >= other) {
+            throw new IllegalArgumentException(other + NOT_LESS_THAN_ERROR_MESSAGE);
+        }
+    }
+
+    public Position move(Direction direction) {
+        checkNotNull(direction);
+        if (direction == LEFT) {
```

Comment:
`direction`에게 메시지를 보내는 방식으로 처리해보는 것도 좋을 것 같아요.
```suggestion
        if (direction.isLeft()) {
```